src/ui/public/vislib/__tests__/components/zero_injection.js

```diff
@@ -419,6 +419,23 @@ describe('Vislib Zero Injection Module Test Suite', function () {
       expect(numberedResults[3]).to.be(4);
       expect(numberedResults[4]).to.be(5);
     });
+
+    it('should return an array of values in the correct order when orderBuckets is set', function () {
```

Comment:
"Correct order" isn't very precise. Let's make this description more explicit, so people reading this will know exactly what to expect of this text:

```javascript
it('should return an array of values ordered by their sum when orderBucketsBySum is true', function () {
```

I think it's worth updating the preceding test's description to provide better contrast:

```javascript
it('should return an array of values ordered by their index by default', function () {
```